<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Undertale QtGraphics</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="lbl_HP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>HP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Health">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Score">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>6</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>score:
0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Enemy">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>100</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/sprites/sans/sansspirte.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Guide">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Keybinds sheet:
H -&gt; Show hitboxes
Arrows -&gt; Move
Esc-&gt; Close</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
